index.bzl

```diff
@@ -0,0 +1,4 @@
+# This is main entrypoint for the npm package users. Should not be used directly.
```

Comment:
this file is not needed since there's webpack/defs.bzl right?

Reply:
I can’t access webpack/defs.bzl file in the userland workspace and I don't know why. I have added a BUILD file to mark the directory as a package but got nowhere. so that's why I have added an index.bzl file that reexports the rules.